core/include/detray/core/type_traits.hpp

```diff
@@ -0,0 +1,27 @@
+/** Detray library, part of the ACTS project (R&D line)
```

Comment:
I think this file should be in `utils` directory

Reply:
Sure, no problem.